### ISSUE TITLE
Bad scrolling performance

### ISSUE THREAD

**Issue by sanyatuning** [reptilex/tesla-style-solar-power-card#66]
Sometimes I experience some lag when scrolling, mostly in lovalace editor mode. [video](https://user-images.githubusercontent.com/2077907/116822395-dbfdbb80-ab7e-11eb-9297-cbea54b25479.mp4)
I suggest using CSS "transform: translate(x,y)" instead of "position: relative; left: x; top: y" or something like this.

Another one: sometimes rendering is very small without any change.
![Screenshot 2021-05-02 19 49 20](https://user-images.githubusercontent.com/2077907/116822532-a3aaad00-ab7f-11eb-89a9-a00cb313967e.png)

**Comment by reptilex** [comment]
Yes. The problem is not only the css but the way I'm calculating and updating. I'm planning on redoing the whole animation with css steps, but will take a while. Currently I'm busy with the visual editor and since this is a hobby ...
